sdk/storage/azure-storage-files-datalake/test/ut/datalake_path_client_test.cpp

```diff
@@ -379,4 +379,14 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  TEST_F(DataLakePathClientTest, PathGetPropertiesOwnerGroupPermissions)
+  {
+    auto properties = m_pathClient->GetProperties().Value;
+    ASSERT_TRUE(properties.Owner.HasValue());
+    ASSERT_TRUE(!properties.Owner.Value().empty());
```

Comment:
The assert at line 386 appears to be testing that Azure::Nullable works correctly - that should probably be in the test cases for Azure::Nullable instead of the test cases for Azure::Storage.

Otherwise it should be sufficient just to check `.HasValue`.

Reply:
@microzchang  Let's remove Line `386`

Reply:
fixed this.